platform/ui/src/contextProviders/DialogProvider.js

```diff
@@ -128,8 +160,14 @@ const DialogProvider = ({ children, service }) => {
         <Draggable
           key={id}
           disabled={!isDraggable}
-          position={position}
-          defaultPosition={lastDialogPosition || defaultPosition}
+          position={
+            centralize
+              ? centerPositions.find(position => position.id === id)
+              : position
+          }
+          defaultPosition={
+            (useLastPosition && lastDialogPosition) || defaultPosition
+          }
```

Comment:
Ah. Part of this is we're keying off `Draggable`'s API.
Their API is likely to be more generic, as it needs to support more use cases.
If we can narrow/simplify it for our needs in this specific situation, we should.

Reply:
If we are conditionally draggable, we could forego wrapping our dialog in `<Draggable>` if we're not draggable? Saves you on some of the odd classnames around `.DraggableItem.draggable`

Reply:
We still need them to be wrapped in Draggable because we are leveraging positioning/events. The odd classNames are basically to show or not the drag icon (dragging => closed hand and drag => openhand).